Sources/ProjectDescription/Environment.swift

```diff
@@ -5,6 +5,16 @@ public struct Environment {
     public enum Value {
         case boolean(Bool)
         case string(String)
+
+        public func getString(default defaultString: String) -> String {
```

Comment:
I forgot to mention that it'd be great to document the methods so that developers get inline documentation when editing their manifest files in Xcode. After that, we can merge the PR

Reply:
Hi @pepibumur 

Sorry for the delay in doing this. Comments should now be added.